hack/testing-olm/test-200-verify-es-metrics-access.sh

```diff
@@ -3,10 +3,14 @@
 # allowed to retrieve metrices from elasticsearch
 set -euo pipefail
 
+KUBECONFIG=${KUBECONFIG:-$HOME/.kube/config}
```

Comment:
do we need to have the `KUBECONFIG` here? the tests worked correctly before

Reply:
Yeah, we need KUBECONFIG here because the new operator sdk has removed its test framework, so we need to set  KUBECONFIG for the `go test` command. Actually, we have done similar change to test-001:

https://github.com/openshift/elasticsearch-operator/blob/909c5a4754ee2ccc484aa8a9cc85b5bf516cf20d/hack/testing-olm/test-001-operator-sdk-e2e.sh#L4